src/transaction_builder.js

```diff
@@ -401,6 +447,18 @@ TransactionBuilder.prototype.sign = function (index, keyPair, redeemScript, hash
 
           break
 
+        case 'segwitpubkeyhash':
```

Comment:
for P2WPKH as P2SH we place the `redeemScript` (short version) in the `scriptSig` but we'll use the long version when generating the signatureHash (hence we added `signatureScript`)